dev/integration_tests/flutter_driver_screenshot_test/test_driver/driver_screenshot_tester.dart

```diff
@@ -0,0 +1,78 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io' show File;
+import 'package:meta/meta.dart';
+import 'package:path/path.dart' as path;
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:collection/collection.dart';
+
+const String _kPathParent = 'test_driver/goldens/';
+
+/// The utility class that helps test cases to tests screenshots with a [FlutterDriver].
+class DriverScreenShotTester {
+  /// Constructs a [DriverScreenShotTester].
+  ///
+  /// All the parameters are required and must not be null.
+  // ignore: always_require_non_null_named_parameters
+  DriverScreenShotTester(
+      {@required this.testName,
+      @required this.driver,
+      @required this.deviceModel})
```

Comment:
Nit: the formatting will be a little nicer if you put a trailing comma after `deviceModel`

Reply:
done